Source/monster.h

```diff
@@ -327,7 +328,9 @@ void decode_enemy(int m, int enemy);
 
 /* data */
 
-extern int opposite[8];
+extern direction left[8];

```

Comment:
should we add a `NUM_DIRECTION` and use it here instead of `8`?
That would make it clearer that this is additional information for Direction.
The same approach was used for `NUM_INVLOC`.

Reply:
we are trying to get rid of the numb enteries and dirrections already has omni as number 8.